packages/build-tools/src/index.ts

```diff
@@ -26,7 +26,10 @@ async function main() {
     let CliArgs = getCliArgs();
     let src = CliArgs.input;
     let outputPath = CliArgs.output;
-    let runtimeArgs = CliArgs.debug ? '--enable-script-debugging' : '';
+    let runtimeArgs = [
+      CliArgs.debug && '--enable-script-debugging',
+      CliArgs.initLocation && `--init-location ${CliArgs.initLocation}`
+    ].filter(Boolean).join(' ');
```

Comment:
This is too clever for my taste (where my taste is informed by the fact that I needed to really analyze how this works to assure myself that it does what it should).

How about this instead? Slightly cleverer than `if` statements and a bit less verbose, but making clearer (to me, at least) what's going on:
```suggestion
    let runtimeArgs = [];
    CliArgs.debug && runtimeArgs.push('--enable-script-debugging');
    CliArgs.initLocation && runtimeArgs.push(`--init-location ${CliArgs.initLocation}`);
```